server/admin_management/api/app/schemas.py

```diff
@@ -45,7 +60,18 @@ class Requester(BaseModel):
 
 
 class TargetUser(Requester):
-    pass
+    """
+    Inherit from the class "Requester". Same as Requester, the TargetUser can
+    be transformed from FamUser db model. However, for new user, the
+    information will not be available from db model. In such case, some
+    properties will be set from the request parameters or request body.
+    Meethod that relys on TargetUser might need to check if it is a new user by
+    checking "is_new_user()".
+    """
+    user_id: Optional[int] = None
```

Comment:
When target_user is a new user (not exists in db), user_id could be None before saving.

Reply:
Hi Ian, I wonder the TargetUser should have user name, user type, business guid as well? cause that's what returned from the `get_target_user_from_id` method?

Reply:
got it, use the Requester as the base model